### ISSUE TITLE
Pkg.test() on Julia 0.5.0 fails (Mac)

### ISSUE THREAD

**Issue by mdpradeep** [JuliaStats/KernelDensity.jl#42]
```
julia> Pkg.test("KernelDensity")
INFO: Testing KernelDensity
Running tests:
 * univariate.jl
ERROR: LoadError: LoadError: ArgumentError: Normal: the condition σ > zero(σ) is not satisfied.
 in macro expansion at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/Distributions/src/utils.jl:6 [inlined]
 in Type at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/Distributions/src/univariate/continuous/normal.jl:28 [inlined]
 in Type at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/Distributions/src/univariate/continuous/normal.jl:32 [inlined]
 in kernel_dist(::Type{Distributions.Normal}, ::Float64) at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/src/univariate.jl:8
 in #kde_lscv#4(::Type{T}, ::Tuple{Float64,Float64}, ::Function, ::Array{Float64,1}, ::FloatRange{Float64}) at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/src/univariate.jl:178
 in (::KernelDensity.#kw##kde_lscv)(::Array{Any,1}, ::KernelDensity.#kde_lscv, ::Array{Float64,1}, ::FloatRange{Float64}) at ./<missing>:0
 in #kde_lscv#7(::Tuple{Float64,Float64}, ::Int64, ::Type{T}, ::Tuple{Float64,Float64}, ::Function, ::Array{Float64,1}) at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/src/univariate.jl:200
 in macro expansion; at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/test/univariate.jl:50 [inlined]
 in anonymous at ./<missing>:?
 in include_from_node1(::String) at ./loading.jl:488
 in include_from_node1(::String) at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/julia/Contents/Resources/julia/lib/julia/sys.dylib:?
 in macro expansion; at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/test/runtests.jl:12 [inlined]
 in anonymous at ./<missing>:?
 in include_from_node1(::String) at ./loading.jl:488
 in include_from_node1(::String) at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/julia/Contents/Resources/julia/lib/julia/sys.dylib:?
 in process_options(::Base.JLOptions) at ./client.jl:262
 in _start() at ./client.jl:318
 in _start() at /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/julia/Contents/Resources/julia/lib/julia/sys.dylib:?
while loading /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/test/univariate.jl, in expression starting on line 17
while loading /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/test/runtests.jl, in expression starting on line 9
====================================================================[ ERROR: KernelDensity ]=====================================================================

failed process: Process(`/Applications/JuliaPro-0.5.0.4.app/Contents/Resources/julia/Contents/Resources/julia/bin/julia -Ccore2 -J/Applications/JuliaPro-0.5.0.4.app/Contents/Resources/julia/Contents/Resources/julia/lib/julia/sys.dylib --compile=yes --depwarn=yes --check-bounds=yes --code-coverage=none --color=yes --compilecache=yes /Applications/JuliaPro-0.5.0.4.app/Contents/Resources/pkgs-0.5.0.4/v0.5/KernelDensity/test/runtests.jl`, ProcessExited(1)) [1]

=================================================================================================================================================================
ERROR: KernelDensity had test errors
 in #test#61(::Bool, ::Function, ::Array{AbstractString,1}) at ./pkg/entry.jl:740
 in (::Base.Pkg.Entry.#kw##test)(::Array{Any,1}, ::Base.Pkg.Entry.#test, ::Array{AbstractString,1}) at ./<missing>:0
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}})() at ./pkg/dir.jl:31
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}}, ::String) at ./file.jl:59
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./pkg/dir.jl:31
 in (::Base.Pkg.Dir.#kw##cd)(::Array{Any,1}, ::Base.Pkg.Dir.#cd, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./<missing>:0
 in #test#3(::Bool, ::Function, ::String, ::Vararg{String,N}) at ./pkg/pkg.jl:258
 in test(::String, ::Vararg{String,N}) at ./pkg/pkg.jl:258
```

**Comment by tkelman** [comment]
This was broken by Distributions 0.11.1.

**Comment by ararslan** [comment]
@tkelman Are you sure? I don't see how any of the changes between Distributions v0.11.0 and v0.11.1 could have made a difference here. https://github.com/JuliaStats/Distributions.jl/compare/v0.11.0...v0.11.1

**Comment by tkelman** [comment]
I can get the specific log diff on the day it started failing, which was the day of that tag - I don't think anything else changed

**Comment by ararslan** [comment]
Weird. Well, I'll look into this regardless.

**Comment by tkelman** [comment]
hm maybe not, pinning to older distributions isn't getting it to pass. will dig up the log

**Comment by tkelman** [comment]
was something else, whoops - https://gist.github.com/anonymous/5a720df229b024eb29e93a26be9ce579
Optim or ForwardDiff or WoodburyMatrices?

**Comment by tkelman** [comment]
okay, it was Optim 0.7. bisecting on metadata turns out to be non-obvious...
```
JULIA_PKGDIR=$HOME/Julia/jlpkgtmp/ julia -e 'Pkg.init()'
cd $HOME/Julia/jlpkgtmp/v0.5/METADATA
JULIA_PKGDIR=$HOME/Julia/jlpkgtmp/ git bisect run julia -e 'run(`git checkout -B tmp`); p="KernelDensity"; try; Pkg.add(p); Pkg.test(p); finally; [Pkg.rm(i) for i in keys(Pkg.installed())]; rm(Pkg.dir(".trash"); recursive=true); end'
```

@pkofod any ideas why Optim 0.7 broke this package?

**Comment by tkelman** [comment]
oh right, does this just need tagging for https://github.com/JuliaStats/KernelDensity.jl/pull/40?

**Comment by pkofod** [comment]
Yes, that PR was intended to fix that error.

**Comment by tkelman** [comment]
this should be fixed by 0.3.2
